<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qpid::LoginDialog</class>
 <widget class="QDialog" name="Qpid::LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>287</width>
    <height>505</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>287</width>
    <height>505</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>QpidPhone Login</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#frameCaption {
	background: rgb(49,169,221);
}

QDialog {
	background: rgb(26,108,149);
}

#labelLogo {
	background: rgb(9,41,57);
	border-radius: 5px;
}

QLineEdit {
	background: rgb(9,41,57);
	border-radius: 5px;
	color: rgb(160,160,160);
}

QPushButton {
	color: rgb(255,255,255);
}

#pushButtonLogin {
	border-radius: 5px;
	background: rgb(102,153,0);
}

#pushButtonLogin:pressed {
	border-radius: 5px;
	background: rgb(100,140,0);
}

#pushButtonCreate {
	border-radius: 5px;
	background: rgb(255,102,0);
}

#pushButtonCreate:pressed {
	border-radius: 5px;
	background: rgb(200,90,0);
}

#pushButtonMinimize {
	border-radius: 0px;
	background: rgb(255,102,0);
}

#pushButtonMinimize:pressed {
	border-radius: 0px;
	background: rgb(200,90,0);
}

#pushButtonClose {
	border-radius: 0px;
	background: rgb(255,102,0);
}

#pushButtonClose:pressed {
	border-radius: 0px;
	background: rgb(200,90,0);
}

#labelSplitter {
	background: rgb(42,118,113);
}

#labelTitle {
	color: rgb(255,255,255);
	font-size: 15px;
	font-weight: bold;
}

#lineEditPassword {
	padding-right: 24px;
}

#pushButtonForget {
	border-image: url(:/images/ui/forget_pwd.png);
}

#pushButtonForget:pressed {
	position: relative;
	top: 2px; left: 2px; /* shift the arrow by 2 px */
}</string>
  </property>
  <widget class="QLabel" name="labelLogo">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>100</y>
     <width>95</width>
     <height>95</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>95</width>
     <height>95</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>95</width>
     <height>95</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Qpid::SmartEdit" name="lineEditAccount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>230</width>
     <height>39</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>39</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>39</height>
    </size>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>Enter your login email address</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Qpid::SmartEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>230</width>
     <height>39</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>39</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>39</height>
    </size>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>Enter your password</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLogin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>230</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCreate">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>450</y>
     <width>230</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>230</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Don't have an account? Register &gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSplitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>285</width>
     <height>5</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>285</width>
     <height>5</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>285</width>
     <height>5</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frameCaption">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>285</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>285</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>32</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Voice Easy</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>68</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="sysPanelContainer">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="sysPanel" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonMinimize">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>18</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>18</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonClose">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>18</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>18</height>
             </size>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>9</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonForget">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>298</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Qpid::SmartEdit</class>
   <extends>QLineEdit</extends>
   <header>smartedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonLogin</tabstop>
  <tabstop>lineEditAccount</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>pushButtonClose</tabstop>
  <tabstop>pushButtonCreate</tabstop>
  <tabstop>pushButtonMinimize</tabstop>
  <tabstop>pushButtonForget</tabstop>
 </tabstops>
 <resources>
  <include location="phone.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>Qpid::LoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMinimize</sender>
   <signal>clicked()</signal>
   <receiver>Qpid::LoginDialog</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>loginStarted()</signal>
  <signal>loginSucceeded()</signal>
  <signal>loginFailed()</signal>
  <slot>login()</slot>
 </slots>
</ui>
